docs/ConverterUsage.md

```diff
@@ -0,0 +1,98 @@
+# Converter Usage & Demo
+
+## Usage
+
+`jfrconv [options] <input> [<input>...] <output>`
+
+The output format specified can be only one at a time for conversion from one format to another.
+
+### Available arguments
+
+```
+Conversion options: 
+  -o --output FORMAT   Output format: <filename> followed by any of the extensions: html, collapsed, pprof, pb.gz
+  
+JFR options:
```

Comment:
This is just a copy of `usage` output which is intentionally concise. It makes sense to describe some options in more detail.

Reply:
Sure Andrei, any specific ones that you want to suggest

Reply:
I have added a bit more details for few options Andrei. Please let me know if the changes address this comment.